internal/venice-common/src/main/java/com/linkedin/venice/writer/update/UpdateBuilderImpl.java

```diff
@@ -136,7 +136,7 @@ Exception validateUpdateRecordIsSerializable(GenericRecord updateRecord) {
       Object unresolvedDatum = serializationException.getUnresolvedDatum();
       String unresolvedDatumType = unresolvedDatum instanceof GenericContainer
           ? ((GenericContainer) unresolvedDatum).getSchema().toString()
-          : unresolvedDatum.getClass().getSimpleName();
+          : unresolvedDatum == null ? "null" : unresolvedDatum.getClass().getSimpleName();
```

Comment:
nit: can we assign `unresolvedDatum == null ? "null" : unresolvedDatum.getClass().getSimpleName();` to a string var ? having 2 ternary operator in one statement kinda reads bad.

Reply:
Sure, I can rework it.

Reply:
Done in the latest commit.